dev/src/v1/firestore_client_config.json

```diff
@@ -2,42 +2,37 @@
   "interfaces": {
     "google.firestore.v1.Firestore": {
       "retry_codes": {
+        "non_idempotent": [],
         "idempotent": [
           "DEADLINE_EXCEEDED",
-          "INTERNAL",
           "UNAVAILABLE"
         ],
-        "non_idempotent": []
+        "d_e_a_d_l_i_n_e_e_x_c_e_e_d_e_d_i_n_t_e_r_n_a_l_u_n_a_v_a_i_l_a_b_l_e": [
+          "DEADLINE_EXCEEDED",
+          "INTERNAL",
+          "UNAVAILABLE"
+        ]
       },
       "retry_params": {
         "default": {
           "initial_retry_delay_millis": 100,
           "retry_delay_multiplier": 1.3,
           "max_retry_delay_millis": 60000,
-          "initial_rpc_timeout_millis": 60000,
-          "rpc_timeout_multiplier": 1.0,
-          "max_rpc_timeout_millis": 60000,
-          "total_timeout_millis": 600000
-        },
-        "streaming": {
-          "initial_retry_delay_millis": 100,
-          "retry_delay_multiplier": 1.3,
-          "max_retry_delay_millis": 60000,
-          "initial_rpc_timeout_millis": 60000,
-          "rpc_timeout_multiplier": 1.0,
-          "max_rpc_timeout_millis": 60000,
+          "initial_rpc_timeout_millis": 20000,
```

Comment:
All timeouts should stay the same (60s)

Reply:
They are coming from `firestore_grpc_service_config.json`, cc: @noahdietz here

Reply:
The `initial_rpc_timeout` is not a supported field anymore with the new config, @alexander-fenster this is one of the default values you decided to set it to. Also, this specific field is not the "RPC timeout" or the total_timeout/timeout_millis. The source for that is the "timeout" field int he grpc_service_config which is currently set to 60s

https://github.com/googleapis/googleapis/blob/b10e4547017ca529ac8d183e839f3c272e1c13de/google/firestore/v1/firestore_grpc_service_config.json